apps/web/screens/OfficialJournalOfIceland/OJOISearch.tsx

```
@@ -414,13 +511,74 @@ const OJOISearch: CustomScreen<OJOISearchProps> = ({
+  }
+
+  if (query.sida && typeof query.sida === 'string') {
+    const check = !isNaN(parseInt(query.sida))
+
+    if (check) {
+      page = parseInt(query.sida)
+    }
+  }
+
+  if (query.pageSize && typeof query.pageSize === 'string') {
+    const check = !isNaN(parseInt(query.pageSize))
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)